driver/src/main/com/mongodb/reactivestreams/client/vault/package-info.java

```diff
@@ -17,6 +17,10 @@
 /**
  * This package contains the Key Vault API
  *
+ * <p>
```

Comment:
I didn't add this to the java driver package-info. Can you add it there as well?

Reply:
done